services/web/client/source/class/osparc/desktop/preferences/pages/OrganizationsPage.js

```diff
@@ -275,6 +281,49 @@ qx.Class.define("osparc.desktop.preferences.pages.OrganizationsPage", {
       });
     },
 
+    __deleteOrganization: function(orgKey) {
+      let org = null;
+      this.__orgsModel.forEach(orgModel => {
+        if (orgModel.getGid() === parseInt(orgKey)) {
+          org = orgModel;
+        }
+      });
+      if (org === null) {
+        return;
+      }
+
+      const name = org.getLabel();
+      const msg = this.tr("Are you sure you want to delete ") + name + "?";
+      const win = new osparc.ui.window.Confirmation(msg);
+      win.center();
+      win.open();
+      win.addListener("close", () => {
+        if (win.getConfirmed()) {
+          const params = {
+            url: {
+              "gid": orgKey
+            }
+          };
+          osparc.data.Resources.fetch("organizations", "delete", params)
+            .then(() => {
+              osparc.store.Store.getInstance().reset("organizations");
```

Comment:
how does this affect all relations to an organization? e.g. classifiers, organization users, etc

Reply:
Those entries in the DB that have the group id as a field are deleted. On the other hand, those other fields where the gid is part of a json object, I'm afraid that will stay orphan.

As you know, the DB requires some love, and once we refactor these kind of entries we will remove the orphan ones.

Reply:
@odeimaiz can you then create a maintenance case to remind us to refaactor/review this please?